src/java/fp/v11/EditIssue.java

```diff
@@ -90,7 +98,7 @@ public static void main( String[] args )
          //msg.writeTo( System.out );
          
          // Make SOAP call
-         SOAPMessage reply = connection.call( msg, "https://localhost/MRcgi/MRWebServices.pl" );
+         SOAPMessage reply = connection.call( msg, "https://localhost/REST/1.0/ticket/edit" );
```

Comment:
@G3nK3n -- now this is confusing; FootPrints does not support REST -- where does this come from?? This is not RT nor BitBucket nor GitHub.